### ISSUE TITLE
Interactive visualization of results

### ISSUE THREAD

**Issue by dganguli** [spacetx/starfish#49]
There currently exists a prototype interactive visualization of processed Starfish data. The code lives here: https://github.com/chanzuckerberg/starfish/tree/master/viz

There are several interesting directions to take this in:
1. Split this into it's own repo
2. Scope out possible UI elements, e.g., right now the visualizer allows one to look at spots, segmentation results, turn on and off genes, and pan/zoom. What are we missing? What could be improved?
3. Have this visualizer connected to the CLI, such that after a CLI user finishes running a pipeline, they can visually inspect results
4. Have this visualizer connected to the API, potentially so that it can be run from a Jupyter notebook
5. Work on ensuring that our standardized file formats work well with the visualizer.

**Comment by dganguli** [comment]
The web-based visualization tool has been moved to: https://github.com/spacetx/starfish-viz. After #390 it will also be removed from starfish
